src/bindings/python/fluxacct/accounting/formatter.py

```diff
@@ -306,3 +306,21 @@ def __init__(self, cursor, username):
         super().__init__(
             cursor, error_msg=f"user {self.username} not found in association_table"
         )
+
+    def list_banks(self):
+        """
+        Return all of the banks that the user belongs to with each bank
+        on its own line.
+
+        Args:
+            username: the name of the user.
```

Comment:
Docstring misrepresents arguments to this function?

Reply:
Thank you for catching this!! I've updated the docstring.